.github/workflows/e2e-tests.yml

```
@@ -42,17 +42,43 @@ jobs:
+        docker compose -f docker-compose.e2e.yml up -d backend frontend redis
         sleep 30
 
     - name: Wait for services to be healthy
       run: |
         timeout 60 bash -c 'until curl -f http://localhost:8000/health; do sleep 2; done'
         timeout 60 bash -c 'until curl -f http://localhost:3000; do sleep 2; done'
 
+    - name: Security checks before E2E tests
+      run: |
+        echo "🔍 Running pre-test security checks..."
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)